openupgrade_scripts/scripts/point_of_sale/14.0.1.0.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,99 @@
+---Models in module 'point_of_sale'---
+---Fields in module 'point_of_sale'---
+point_of_sale / pos.config               / iface_precompute_cash (boolean): DEL
+point_of_sale / pos.config               / manage_orders (boolean)       : NEW
+point_of_sale / pos.config               / manual_discount (boolean)     : NEW hasdefault
+point_of_sale / pos.config               / module_pos_reprint (boolean)  : DEL
+# NOTHING TO DO
+
+point_of_sale / pos.config               / picking_type_id (many2one)    : now required, req_default: function
+# DONE: pre-migration fill record having empty field with default value
+
+point_of_sale / pos.config               / product_configurator (boolean): NEW
+point_of_sale / pos.order                / _order                        : _order is now 'date_order desc, name desc, id desc' ('id desc')
+point_of_sale / pos.order                / is_tipped (boolean)           : NEW
+# NOTHING TO DO
+
+point_of_sale / pos.order                / location_id (many2one)        : DEL relation: stock.location
+# NOTHING TO DO
+
+point_of_sale / pos.order                / picking_id (many2one)         : DEL relation: stock.picking
+point_of_sale / pos.order                / picking_ids (one2many)        : NEW relation: stock.picking
+# DONE: post-migration convert data many2one to one2many
+
+point_of_sale / pos.order                / tip_amount (float)            : NEW
+point_of_sale / pos.order.line           / full_product_name (char)      : NEW
+point_of_sale / pos.payment              / cardholder_name (char)        : NEW
+point_of_sale / pos.payment              / is_change (boolean)           : NEW hasdefault
+point_of_sale / pos.payment              / payment_status (char)         : NEW
+point_of_sale / pos.payment              / ticket (char)                 : NEW
+point_of_sale / pos.payment.method       / active (boolean)              : NEW hasdefault
+point_of_sale / pos.session              / cash_real_difference (float)  : NEW
+point_of_sale / pos.session              / cash_real_expected (float)    : NEW
+point_of_sale / pos.session              / cash_real_transaction (float) : NEW
+point_of_sale / pos.session              / picking_ids (one2many)        : NEW relation: stock.picking
+point_of_sale / pos.session              / state (selection)             : selection_keys is now '['closed', 'closing_control', 'opened', 'opening_control']' ('['closed', 'closing_control', 'new_session', 'opened', 'opening_control']')
```

Comment:
@hieu1211 i think need to map 'new_session' to 'opening_control' / 'opened'

Reply:
Yeah, new_session has to be mapped to one of the other options.

Reply:
i missed it, thanks!